resolwe_bio/processes/fusion_calling/arriba.py

```diff
@@ -0,0 +1,126 @@
+"""Gene fusion detection with Arriba."""
+
+import pandas as pd
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, Process, SchedulingClass
+
+
+def get_contig_names(gtf_file):
+    """Get unique contig names.
+
+    List of contig (chromosome) names is required by arriba (parameter -i).
+    This function covers possible edge cases where contig names are not common.
+    """
+
+    gtf = pd.read_csv(gtf_file, sep="\t", header=None, usecols=[0])
+    contigs = set(gtf[0])
+    out = " ".join(list(contigs))
+    return out
+
+
+class Arriba(Process):
+    """Run Arriba for fusion detection from RNA-Seq data.
+
+    Arriba is a command-line tool for gene fusion detection from RNA-Seq data.
+    This process detects gene fusions from RNA-Seq data using the Arriba tool.
+    The input can be a BAM file from the STAR aligner, and additional optional
+    inputs such as blacklist, protein domains, and known fusions files can be provided.
+    More information about Arriba can be found in the
+    [Arriba manual](https://arriba.readthedocs.io/en/latest/) and in
+    the [original paper](https://genome.cshlp.org/content/31/3/448)
+    The current version of Arriba is 2.4.0.
```

Comment:
Let's see how long it takes us to have this text about version out of sync with the docker image. 😁

Reply:
At least until the next rnaseq docker image update :D